src/encoded/types/transgenic_enhancer_experiment.py

```diff
@@ -49,6 +49,7 @@ class TransgenicEnhancerExperiment(
         'biosamples.biosample_ontology',
         'biosamples.organism',
         'biosamples.characterizations',
+        'biosamples.treatments',
```

Comment:
Do we have treatments in TransgeneicEnhancerExperiments?

Reply:
no we don't but I think theoretically they could be assigned, then I want them embedded